index.html

```diff
@@ -171,10 +171,11 @@ <h2 id="contact-form-desc">
         </h2>
     </div>
     <div class="form">
-        <form action="https://formspree.io/f/mpznrkvl" method="post">
+        <form id="form-id" action="https://formspree.io/f/mpznrkvl" method="post">
             <div class="Names">
             <input type="text" id="fullname" inputname="fullname" maxlength="30" placeholder="Full Name" required><br><input type="text" id="fullnames" inputname="fullname" maxlength="30" placeholder="Last Name" required><br></div>
             <input type="email" id="email" inputname="" placeholder="Email Address" required>
+            <small></small>
             <textarea name="message" id="textarea" placeholder="Enter Text Here" maxlength="500" required></textarea>
             <input type="submit" value="Get in touch" id="get-in-btn" class="standard-button">
         </form>
```

Comment:
[OPTIONAL] The form inputs should be able to clear after clicking the submit button in conformity to best practices.